src/librustc_trans/partitioning.rs

```diff
@@ -102,21 +102,21 @@
 //! source-level module, functions from the same module will be available for
 //! inlining, even when they are not marked #[inline].
 
-use collector::InliningMap;
-use common;
+use rustc_mir::monomorphize::collector::InliningMap;
```

Comment:
Hmmm, doesn't this entire file (`partitioning.rs`) belong in `rustc_mir/monomorphize`?